ide/plugins/java-format/src/content/formats/webdriver-testng.js

```diff
@@ -0,0 +1,523 @@
+        indents(3) + "String alertText = alert.getText();\n" +
+        indents(3) + "if (acceptNextAlert) {\n" +
+        indents(4) + "alert.accept();\n" +
+        indents(3) + "} else {\n" +
+        indents(4) + "alert.dismiss();\n" +
+        indents(3) + "}\n" +
+        indents(3) + "return alertText;\n" +
+        indents(2) + "} finally {\n" +
+        indents(3) + "acceptNextAlert = true;\n" +
+        indents(2) + "}\n" +
+        indents(1) + "}\n" +
+        indents(0) + "}\n";
+
+this.configForm =
+    '<description>Variable for Selenium instance</description>' +
+        '<textbox id="options_receiver" />' +
+        '<description>Package</description>' +
+        '<textbox id="options_packageName" />' +
+        '<description>Header</description>' +
+        '<textbox id="options_header" multiline="true" flex="1" rows="4"/>' +
+        '<description>Footer</description>' +
+        '<textbox id="options_footer" multiline="true" flex="1" rows="4"/>' +
+        '<description>Indent</description>' +
+        '<menulist id="options_indent"><menupopup>' +
+        '<menuitem label="Tab" value="tab"/>' +
+        '<menuitem label="1 space" value="1"/>' +
+        '<menuitem label="2 spaces" value="2"/>' +
+        '<menuitem label="3 spaces" value="3"/>' +
+        '<menuitem label="4 spaces" value="4"/>' +
+        '<menuitem label="5 spaces" value="5"/>' +
+        '<menuitem label="6 spaces" value="6"/>' +
+        '<menuitem label="7 spaces" value="7"/>' +
+        '<menuitem label="8 spaces" value="8"/>' +
+        '</menupopup></menulist>' +
+        '<checkbox id="options_showSelenese" label="Show Selenese"/>';
+
+this.name = "TestNG (WebDriver)";
+this.testcaseExtension = ".java";
+this.suiteExtension = ".xml";
+this.webdriver = true;
+
+WDAPI.Driver = function() {
+  this.ref = options.receiver;
+};
+
+WDAPI.Driver.searchContext = function(locatorType, locator) {
+  var locatorString = xlateArgument(locator);
+  switch (locatorType) {
+    case 'xpath':
+      return 'By.xpath(' + locatorString + ')';
+    case 'css':
+      return 'By.cssSelector(' + locatorString + ')';
+    case 'id':
+      return 'By.id(' + locatorString + ')';
+    case 'link':
+      return 'By.linkText(' + locatorString + ')';
+    case 'name':
+      return 'By.name(' + locatorString + ')';
+    case 'tag_name':
+      return 'By.tagName(' + locatorString + ')';
+  }
+  throw 'Error: unknown strategy [' + locatorType + '] for locator [' + locator + ']';
+};
+
+WDAPI.Driver.prototype.back = function() {
+  return this.ref + ".navigate().back()";
+};
+
+WDAPI.Driver.prototype.close = function() {
+  return this.ref + ".close()";
+};
+
+WDAPI.Driver.prototype.findElement = function(locatorType, locator) {
+  return new WDAPI.Element(this.ref + ".findElement(" + WDAPI.Driver.searchContext(locatorType, locator) + ")");
+};
+
+WDAPI.Driver.prototype.findElements = function(locatorType, locator) {
```

Comment:
Formatting issues ->
function(locatorType, locator) { should be like this -> function (locatorType, locator) {